docs/policy.json.5.md

```diff
@@ -2,15 +2,18 @@
 % Miloslav Trmač
 % September 2016
 
-# Signature verification policy file format
+# NAME
+policy.json - Syntax for the Signature Verification Policy File
```

Comment:
`man 7 man-pages` says
> All words in this line (including the word immediately following the "\-") should be in lowercase, except where English or technical terminological convention dictates otherwise.

Reply:
Fixed, I bet we have a lot of man pages that fail that test.